shared/teams/routes.js

```diff
@@ -71,7 +72,13 @@ const routeTree = makeRouteDefNode({
           component: Member,
         },
         addPeople: {
-          children: {},
+          children: {
+            controlledRolePicker: {
```

Comment:
🐑  move this to a const above so you can just do `children: {controlledRolePicker}` since we may want this in other places to.